pkg/resources/configmaps/configmap.go

```diff
@@ -0,0 +1,46 @@
+package configmaps
+
+import (
+	"errors"
+
+	"github.com/artemiscloud/activemq-artemis-operator/pkg/resources"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func RetrieveConfigMap(namespace string, configMapName string, client client.Client) (*corev1.ConfigMap, error) {
+	if client == nil {
+		// this could happen in some tests
+		return nil, errors.New("caller doesn't provide a client")
```

Comment:
I wonder if this can still happen now that the presence is explicit in extraMounts, it seems odd that it could

Reply:
I'll check it out.

Reply:
yep no longer needed.